azure-pipelines.yml

```diff
@@ -53,7 +53,7 @@ stages:
   jobs:
   - template: .azure-pipelines/azure-pipelines-build.yml
     parameters:
-      buildOptions: 'USERNAME=admin SONIC_BUILD_JOBS=$(nproc) ${{ variables.VERSION_CONTROL_OPTIONS }}'
+      buildOptions: 'USERNAME=admin SONIC_BUILD_JOBS=$(nproc) BUILD_MULTIASIC_KVM=y ${{ variables.VERSION_CONTROL_OPTIONS }}'
```

Comment:
If this option is added, will this pipeline build single-asic VS image plus multi-asic VS image?

Reply:
Yes, it will build single and multi-asic images.
Azure.sonic-buildimage (BuildVS vs) Successful in 211m — BuildVS vs succeeded - this job artifact will have :
sonic-4asic-vs.img.gz (this will be predominantly used in VS testbed as it is lighter than 6-asic image)
sonic-6asic-vs.img.gz 
sonic-vs.img.gz
sonic-vs.bin
This is a pipeline that already uses this build parameter: https://dev.azure.com/mssonic/build/_build?definitionId=342&_a=summary 
Background work is done in this PR: https://github.com/Azure/sonic-buildimage/pull/8270 to support building all KVM images.

Reply:
Then I am good with this change.